src/BaGet.Aliyun/AliyunOSSService.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Aliyun.OSS;
+using BaGet.Aliyun.Configuration;
+using BaGet.Core;
+using Microsoft.Extensions.Options;
+
+namespace BaGet.Aliyun
+{
+    public class AliyunOSSService : IStorageService
```

Comment:
What do you think of the name `AliyunStorageService`? I'd rather avoid acronyms if possible

```suggestion
    public class AliyunOSSService : IStorageService
```